portal-api/src/v1/roles/validate-user-has-sufficient-role.js

```diff
@@ -0,0 +1,18 @@
+const { userHasSufficientRole } = require('./user-has-sufficient-role');
+
+const validateUserHasSufficientRole = ({
```

Comment:
Should this function be renamed to `validateUserIsNonAdminRole`?

Reply:
It doesn't only check that the user is one of the non-admin roles - it checks if they are one of the allowed non-admin roles OR if they are the "I can access any route" role - which is "ukef_operations" in this PR but changes to "admin" (which makes the naming make more sense) in #1975